chromium_src/BUILD.gn

```diff
@@ -74,8 +74,20 @@ static_library("chrome") {
   if (is_linux) {
     sources += [ "//chrome/browser/icon_loader_auralinux.cc" ]
     sources += [
+      "//chrome/browser/certificate_manager_model.cc",
+      "//chrome/browser/certificate_manager_model.h",
       "//chrome/browser/extensions/global_shortcut_listener_x11.cc",
       "//chrome/browser/extensions/global_shortcut_listener_x11.h",
+      "//chrome/browser/net/nss_context.h",
+      "//chrome/browser/net/nss_context_linux.cc",
+      "//chrome/browser/ui/browser_dialogs.cc",
+      "//chrome/browser/ui/browser_dialogs.h",
```

Comment:
it's surprising to me that these are needed, what requires them?

Reply:
@nornagon it's an unfortunate little undefined symbol chain - `certificate_manager_model.cc` requires `GetNSSCertDatabaseForResourceContext` (defined in `nss_context_linux.cc`) as well as `chrome::UnlockSlotsIfNecessary` (defined in `crypto_module_password_dialog_nss.cc`) so i aded those, which in turn introduced a few more undefined symbols, requiring the last several files be added.

Reply:
I will look at reducing the dependency chain  before merging this PR, if you don't mind @codebytere . We should be able to omit these by patching some code in `certificate_manager_model`

Reply:
Sounds good - i was avoiding extra patch surface but if we think that's a better alternative i'll swap to that :)